src/Modifier/StreamIterator.php

```diff
@@ -156,18 +156,17 @@ public function setFlags($flags)
      * @param array  $fields
      * @param string $delimiter
      * @param string $enclosure
-     * @param string $escape
```

Comment:
You removed the `$escape` type hint, but the param is still available in the method.